easybuild/easyconfigs/c/CellProfiler/CellProfiler-4.2.4-foss-2021a.eb

```diff
@@ -0,0 +1,116 @@
+# Author: J. Sassmannshausen (Imperial College London/UK)
+
+easyblock = 'PythonBundle'
+
+name = 'CellProfiler'
+version = '4.2.4'
+
+homepage = 'https://github.com/CellProfiler/CellProfiler'
+description = """CellProfiler is a free open-source software designed to enable 
+biologists without training in computer vision or programming to quantitatively 
+measure phenotypes from thousands of images automatically."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+dependencies = [
+    ('Python', '3.9.5'),
+    ('SciPy-bundle', '2021.05'),
+    ('boto3', '1.20.13'),
+    ('h5py', '3.2.1'),
+    ('imageio', '2.10.5'),
+    ('Pillow', '8.2.0'),
+    ('PyZMQ', '22.3.0'),
+    ('scikit-image', '0.18.3'),
+    ('scikit-learn', '0.24.2'),
+    ('wxPython', '4.1.0'),
```

Comment:
@sassy-crick This should be bumped to `4.1.1`, see #15046 (and then #16523 should be closed)?

Reply:
Done. Thanks for spotting this.